code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -936,3 +936,424 @@ Basically, we fill the time between now and 2s from now with hands based off the
+
+/datum/reagent/inverse/baldium
+	name = "Maldium"
+	description = "Potent psychotropic that causes intense anger within users."
+	color = "#ff0000"
+	ph = 1
+	metabolization_rate = 0.4 * REM
+	tox_damage = 0
+	var/delayed_burn_damage = 0
+
+/datum/reagent/inverse/baldium/on_mob_metabolize(mob/living/carbon/affected_mob)
+	. = ..()
+	affected_mob.add_shared_particles(/particles/smoke/steam/mald)
+	affected_mob.manual_emote("inhales sharply.")
+	to_chat(affected_mob, span_warning("You can't help but find everything more irritating."))
+
+/datum/reagent/inverse/baldium/on_mob_end_metabolize(mob/living/carbon/affected_mob)
+	. = ..()
+	affected_mob.remove_atom_colour(ADMIN_COLOUR_PRIORITY, "#fe0000")
+	affected_mob.remove_shared_particles(/particles/smoke/steam/mald)
+	affected_mob.manual_emote("exhales sharply.")
+	to_chat(affected_mob, span_warning("You feel an intense burning sensation as your anger subsides!"))
+
+/datum/reagent/inverse/baldium/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	. = ..()
+	delayed_burn_damage += (seconds_per_tick * 1)
+	if(holder.has_reagent(/datum/reagent/consumable/salt))
+		holder.remove_reagent(/datum/reagent/inverse/baldium, 3 * REM * seconds_per_tick)
+		holder.remove_reagent(/datum/reagent/consumable/salt, 1 * REM * seconds_per_tick)
+
+	switch(current_cycle)
+		if(5)
+			affected_mob.add_atom_colour("#fe0000", ADMIN_COLOUR_PRIORITY)
+			if(!HAS_TRAIT(affected_mob, TRAIT_PACIFISM))
+				var/turf/angery_blast = get_turf(affected_mob)
+				goonchem_vortex(angery_blast, 1, 4)
+				to_chat(affected_mob, span_warning("You can't control yourself as you yell out ANGRILY!!"))
+				affected_mob.emote("scream")
+				affected_mob.say(pick("RAGE!! UNLEASH THE RAGE!!","I'M SO ANGRY!!","WHY I OUGHTA...","AAARRRGGHH!!"), forced = type)
+			else
+				to_chat(affected_mob, span_warning("You calmly yet firmly state your discontent."))
+				affected_mob.say(pick("Aaahhhh...","Rage, woo yeah...","I'm slightly upset...",), forced = type)
+
+		if(6 to INFINITY)
+			if(SPT_PROB(10, seconds_per_tick))
+				affected_mob.manual_emote(pick("breathes rapidly!","huffs, and puffs...","stares MENACINGLY!","sighs AGGRESSIVELY!","sheds a tear ANGRILY!"))
+
+/datum/reagent/inverse/baldium/on_mob_delete(mob/living/affected_mob)
+	. = ..()
+	affected_mob.log_message("has taken [delayed_burn_damage] burn damage from maldium's aftereffects", LOG_ATTACK)
+	affected_mob.adjustFireLoss(delayed_burn_damage, updating_health = TRUE, required_bodytype = affected_bodytype)
+
+/datum/reagent/inverse/colorful_reagent
+	name = "Dulling Reagent"
+	description = "Extremely drab coloring pigment that is favored by corporations who wish to maximize suffering."
+	color = COLOR_GRAY
+	var/list/random_color_list = list("#1a1a1a","#2e2e2e","#424242","#565656","#6a6a6a","#7e7e7e","#929292","#a6a6a6","#bababa","#cecece")
```

Comment:
Please move all your new vars to be below the existing ones (and give them documenting comments)